style.css

```diff
@@ -176,4 +176,126 @@ select {
   align-items: center;
   padding: 0 var(--padding);
   border-top: var(--border);
+}
+
+
+
+/*
+NAVIGATION STYLES
+*/
+
+.dpc-nav {
+  font-family: Sans;
+}
+.dpc-nav ul {
+  display: flex;
+  align-items: center;
+}
+
+.dpc-nav * {
+  text-decoration: none;
+  font-size: 1rem;
+  font-weight: 300;
+  text-transform: uppercase;
+}
+
+.dpc-nav li a:hover {
+  background-color: #FAC05E;
+  border-radius: 4px;
+}
+
+ /* .dpc-nav li+li {
+  margin-left: 20px;
+}  */
+
+
+/*About Us Styles 
+*/
+.section {
+  background: #E5E5E5;
+}
+.About {
+  text-align:center;
+}
+
+.missions{
+  padding: 2rem; 
+  text-align:center;
+  font-size: 1.4rem;
+line-height: 28px;
+}
+.Who h1{
+  font-size: 4.5rem;
+  margin-top: 5rem;
+  color: #93C0A4;
+  font-style: normal;
+  font-weight: normal;
+
+}
+
+.Who p{
+  font-weight: 600;
+  line-height: 23px;
+  font-size: 1.4rem;
+  margin-top: 1rem;
+  color: #EB9B14;
+}
+.center {
+    margin-top: 3rem;
+    border-radius: 20px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    height: 22.125rem;
+    width: 83.9375rem; 
+   
+}
+.Containers {
+  box-sizing: border-box;
+  /* justify-content: space-between; */
+  justify-content:center;
+  display: flex;
+  flex-direction: row;
+}
+
+#box1 {
```

Comment:
its hard to understand where box1, box2, box3 would be used. Are there maybe some more descriptive names?